src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/WriteableBitmap_MultiInvalidate.xaml.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Reflection;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace UITests.Windows_UI_Xaml_Controls.ImageTests
+{
+	[Sample("WriteableBitmap")]
+	public sealed partial class WriteableBitmap_MultiInvalidate : Page
+	{
+		private readonly Random _randomizer = new Random();
```

Comment:
Random, but with a fixed seed, if it's invalidation that tested, increment the seed on each update so we can validate stable behavior with screenshots.

```suggestion
		private readonly Random _randomizer = new Random(0);
```

Reply:
I think the seed should be `42`, not `0` ;-)

Reply:
This 👆👆👆